Energy Saver/Pages/Statistics.cshtml.cs

```diff
@@ -1,32 +1,23 @@
-using ChartJSCore.Helpers;
 using ChartJSCore.Models;
 using Energy_Saver.DataSpace;
 using Energy_Saver.Model;
 using Energy_Saver.Services;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using Microsoft.EntityFrameworkCore;
 using System.Security.Claims;
 using static Energy_Saver.Model.Serialization;
+using static Energy_Saver.Services.ChartService;
 
 namespace Energy_Saver.Pages
 {
     public class StatisticsModel : PageModel
     {
         private readonly IChartService _chartService;
+        public Chart? YearChart { get; set; }
+        public Chart? MonthChart { get; set; }
+        public List<List<Taxes>>? Taxes { get; set; }
```

Comment:
Should use List<Taxes> instead of List<List<Taxes>>

Reply:
Will refactor for next week